Makefile

```diff
@@ -35,11 +35,9 @@ fmt: ## Format code with black and isort
 
 .PHONY: lint
 lint: ## Run linters
-	mypy flytekit/core || true
-	mypy flytekit/types || true
-	mypy tests/flytekit/unit/core || true
-	# Exclude setup.py to fix error: Duplicate module named "setup"
-	mypy plugins --exclude setup.py || true
+	mypy flytekit/core
+	mypy flytekit/types
+	mypy --allow-empty-bodies --allow-redefinition tests/flytekit/unit/core
```

Comment:
why do we need those flags?

Reply:
In flytekit tests, many functions' body is empty, like [this](https://github.com/flyteorg/flytekit/blob/8c6764c299427cc5e4bda4406b2396082e099582/tests/flytekit/unit/core/test_docstring.py#L31), so we add `allow-empty-bodies` here.
I removed allow-redefinition, and updated the comment as well.